helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -139,7 +139,7 @@ spec:
     rules:
     - alert: LokiCompactorFailedCompaction
       annotations:
-        dashboard: loki-retention/loki-retention
+        dashboard: retention/loki-retention
```

Comment:
Nope that's incorrect :D Can you restart grafana to have the upgraded mixins?

Reply:
You're right :facepalm: 
It the missing opsrecipe the same, just a problem of refreshing something? :grinning:

Reply:
Nope, I did not know how to fix it

Reply:
because logs say nothing. I guess it's out of space though